vision_msgs/Detection2D detection
rectbot_cv/KeyPoint[] keypoints